samples/can/back_and_forth/main.cpp

```diff
@@ -50,6 +50,8 @@ int main() {
     if (result != IO::CAN::CANStatus::OK) {
         uart.printf("Failed to connect to the CAN network\r\n");
         return 1;
+    } else {
+        uart.printf("Connection Successful!\r\n");
```

Comment:
Else statement is redundant here. If the CANStatus is not ok, the printf statement would never reach anyways

Reply:
Is there any detriment to having it? I see how it is redundant but it could be helpful if you are looking for debug information.

Reply:
Not really a detriment besides just being a style issue. The compiler would optimize the same anyways. Not understanding fully what you mean by looking for debug information.

Reply:
Issue should be resolved. I was using that as tester code to check if everything was up and running. Should be good now, if not let me know.